### ISSUE TITLE
Questions about training delay model and transductive model in LOSTIN

### ISSUE THREAD

**Issue by YiKangOY** [lydiawunan/LOSTIN#1]
Hi, 

I have run your GNN+LSTM code and had a few questions:
1.	I tried with GNN+LSTM delay model (2 layers of GIN) by both changing the **data_folder** in **main_gnn_customized_area.py (emb_dim=20)** to lstm/delay and using the **main_gnn_customized_delay.py (emb_dim=10)**. I tried for several times, but the reporting MAPE is ranging from 15% to 20%, which is far lower than the value in the paper(2.5%). The minimum validation loss I can get is about 0.001665 and the minimum training loss I can get is about 0.1. The training epochs is the same as that in paper, 20. 

2.	I notice the datasets in the GNN+LSTM directory, **lstm/data_area** and **lstm/data_delay**, they both contain 11 designs in training, valid and test set. I select the data points belonging to design a0 to a5 that correspond to the transductive scenario in your paper. I split the training, valid and test set by 20-5-75. Then I trained with area model and delay model for 20 epochs and test them. The MAPE for delay is 14.7 while the MAPE for area is 18.8%.

I am wondering if I made some mistakes when training the delay model. I would be more than grateful if you could provide me with some hints. If you need any training logs, please feel free to tell me!

BTW, I notice there may be a problematic datapoint for design a8 where both the delay and area are 0.0.

Thanks for your time!
